<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connect4App</class>
 <widget class="QMainWindow" name="connect4App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>connect4App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>711</width>
      <height>451</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="space_1_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_1_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_1_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_1_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_1_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_1_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_2_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>320</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_2_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_2_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>260</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_2_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>390</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_2_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_2_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_3_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>320</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_3_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_3_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_3_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>390</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_3_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_3_1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_4_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>320</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_4_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_4_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_4_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>390</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_4_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_4_1">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_5_5">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>320</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_5_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_5_4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>260</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_5_6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>390</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_5_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>110</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_5_1">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_6_5">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>320</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_6_3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_6_4">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>260</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_6_6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>390</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_6_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>110</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_6_1">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_7_5">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>320</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_7_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>190</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_7_4">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>260</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_7_6">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>390</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_7_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>110</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
    <widget class="QLabel" name="space_7_1">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop Piece</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop Piece</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop Piece</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop Piece</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop Piece</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop Piece</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop Piece</string>
    </property>
   </widget>
   <widget class="QPushButton" name="instructionsButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Instructions</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose_Application_2"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose_Application">
   <property name="text">
    <string>Close Application</string>
   </property>
  </action>
  <action name="actionClose_Application_2">
   <property name="text">
    <string>Close Application</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
